app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class='login-container'>
+  <h2>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Username" %>
+    </div>
+
+    <div class="field">
+      <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field rememberable">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="actions">
+      <%= f.submit "Log in" %>
+    </div>
+  <% end %>
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] @dev89s and @akos29 Great work adding the login form, however, the design doesn't match the wireframe given, below is a comparison, with highlights on the differences.


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/dev89s/blog-app/assets/61408860/4e92b14b-80d6-4537-b258-756a1865203d)  | ![image](https://github.com/dev89s/blog-app/assets/61408860/bc762f3b-4ba3-4d22-914f-9647d2bccede)